src/fmeval/transforms/summarization_accuracy_metrics.py

```diff
@@ -36,67 +37,86 @@ class SummarizationAccuracyMetric(Transform):
 
     def __init__(
         self,
-        target_output_keys: List[str],
-        model_output_keys: List[str],
         output_keys: List[str],
+        model_output_keys: List[str],
         allow_duplicate_input_keys: bool,
+        target_output_keys: Optional[List[str]] = None,
```

Comment:
Let's keep this as a positional argument so that 1) the argument order doesn't change, which would break things for existing users, and 2) the default configuration isn't invalid; currently, if you use the default options, you'll get a validation error.

For point 1), I really mean that we don't want the signatures of the concrete subclasses to change. Obviously, no one is directly calling this ABC's init method.